OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -768,121 +746,93 @@ constant ConfigFlag POST_OPT_MODULES = CONFIG_FLAG(16, "postOptModules",
     ("solveLinearSystem", Gettext.notrans("solve linear system with newton step")),
     ("solveSimpleEquations", Gettext.notrans("Solves simple equations")),
     ("symSolver", Gettext.notrans("Rewrites the ode system for implicit Euler method. This module requires +symSolver.")),
-    ("symbolicJacobian", Gettext.notrans("Detects the sparse pattern of the ODE system and calculates also the symbolic Jacobian if flag '--generateSymbolicJacobian' is enabled.")),
+    ("symbolicJacobian", Gettext.notrans("Detects the sparse pattern of the ODE system and calculates also the symbolic Jacobian if flag '--generateDynamicJacobian=symbolic'.")),
```

Comment:
Only ODE or also DAE?

Reply:
Also elsewhere in this file...

Reply:
Only ODE. There are special ones for DAE:
https://github.com/OpenModelica/OpenModelica/blob/1711781d8ab93da9e5ba1c8466b537da28db838b/OMCompiler/Compiler/Util/Flags.mo#L1248-L1263